Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM gcr.io/distroless/java17
 WORKDIR /app
 COPY build/libs/app.jar app.jar
-ENV JDK_JAVA_OPTIONS="-Dlogback.configurationFile=logback.xml"
+ENV JDK_JAVA_OPTIONS="-XX:MaxRAMPercentage=75 -Dlogback.configurationFile=logback.xml"
```

Comment:
Noe begrunnelse til akkurat 75? Eller må vi kanskje bare teste og følge med?

Reply:
Det er dette vi har brukt i en del andre app'er også (ser at noen navikt-repo's bruker 90% også, men de fleste ligger på 75). Det viktigste er å sette denne ganske høyt, for default i Java VM'en er bare 25% - og da er det jo mye minne som bare blir stående ubrukt i Kubernetes-pod'en.